components/servo/webview_delegate.rs

```diff
@@ -515,12 +516,20 @@ pub trait WebViewDelegate {
     }
 
     /// Show a context menu to the user
-    fn show_context_menu(
+    fn show_native_context_menu(
         &self,
         _webview: WebView,
         result_sender: GenericSender<ContextMenuResult>,
-        _: Option<String>,
-        _: Vec<String>,
+        _options: NativeContextMenu,
+    ) {
+        let _ = result_sender.send(ContextMenuResult::Ignored);
+    }
+
+    fn show_custom_context_menu(
+        &self,
+        _webview: WebView,
+        result_sender: GenericSender<ContextMenuResult>,
+        _items: Vec<ContextMenuItem>,
     ) {
```

Comment:
What is the difference between a native and custom context menu?

Reply:
Native menu is the one the browser will display by default, with actions that can only be handled by the browser itself. Custom context menus are created by JS.

Reply:
On second thought, I'm considering removing that functionality.